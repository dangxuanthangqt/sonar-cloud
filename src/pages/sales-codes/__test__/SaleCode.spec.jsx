import React from 'react';
import { render, screen } from '../../../common/utils/testUtils';
import SalesCodes from '@/pages/sales-codes';

describe('Data source', () => {
  it('should match snapshot', async () => {
    const { container } = render(<SalesCodes />);
    expect(container).toMatchSnapshot();
  });

  it('should match element in document', async () => {
    const { getByText, getByRole } = render(<SalesCodes />);

    expect(
      screen.getByRole('button', { name: 'Previous' })
    ).toBeInTheDocument();

    expect(
      screen.getByRole('button', { name: 'Add New Row' })
    ).toBeInTheDocument();

    expect(screen.getByRole('textbox', { name: '' })).toBeInTheDocument();

    expect(screen.getByText('Datasources')).toBeInTheDocument();
  });
});
