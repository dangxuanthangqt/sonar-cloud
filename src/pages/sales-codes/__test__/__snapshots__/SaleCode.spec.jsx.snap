// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Data source should match snapshot 1`] = `
<div>
  <main>
    <div
      class="makeStyles-container-9 MuiBox-root css-0"
    >
      <div
        class="flex justify-center flex-col items-center"
      >
        <button
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium makeStyles-iconPrevious-16 css-v3imdt-MuiButtonBase-root-MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-17slstj-MuiSvgIcon-root"
            data-testid="NavigateBeforeRoundedIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M14.91 6.71a.9959.9959 0 0 0-1.41 0L8.91 11.3c-.39.39-.39 1.02 0 1.41l4.59 4.59c.39.39 1.02.39 1.41 0 .39-.39.39-1.02 0-1.41L11.03 12l3.88-3.88c.38-.39.38-1.03 0-1.41z"
            />
          </svg>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <p
          class="makeStyles-titleBtnPrevious-17"
          style="display: none;"
        >
          Previous
        </p>
      </div>
      <div
        class="MuiStepper-root MuiStepper-horizontal makeStyles-stepper-18 step-form-wrapper css-m5vj9m-MuiStepper-root"
      >
        <div
          class="MuiStep-root MuiStep-horizontal makeStyles-step-19 css-1bw0nnu-MuiStep-root"
        >
          <span
            class="makeStyles-stepLabelRoot-10 MuiStepLabel-root MuiStepLabel-horizontal css-ascpo7-MuiStepLabel-root"
          >
            <span
              class="makeStyles-iconStyle-11 MuiStepLabel-iconContainer css-vnkopk-MuiStepLabel-iconContainer"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiStepIcon-root Mui-active css-1ktinih-MuiSvgIcon-root-MuiStepIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <circle
                  cx="12"
                  cy="12"
                  r="12"
                />
                <text
                  class="MuiStepIcon-text css-ighunu-MuiStepIcon-text"
                  dominant-baseline="central"
                  text-anchor="middle"
                  x="12"
                  y="12"
                >
                  1
                </text>
              </svg>
            </span>
            <span
              class="MuiStepLabel-labelContainer css-16ubnlw-MuiStepLabel-labelContainer"
            >
              <span
                class="makeStyles-labelStyle1-12 MuiStepLabel-label makeStyles-activeStep-13 Mui-active css-gv1taf-MuiStepLabel-label"
              >
                Datasources
              </span>
            </span>
          </span>
        </div>
        <div
          class="MuiStepConnector-root MuiStepConnector-horizontal Mui-disabled css-j5w0w9-MuiStepConnector-root"
        >
          <span
            class="MuiStepConnector-line MuiStepConnector-lineHorizontal css-z7uhs0-MuiStepConnector-line"
          />
        </div>
        <div
          class="MuiStep-root MuiStep-horizontal makeStyles-step-19 css-1bw0nnu-MuiStep-root"
        >
          <span
            class="makeStyles-stepLabelRoot-10 MuiStepLabel-root MuiStepLabel-horizontal Mui-disabled css-ascpo7-MuiStepLabel-root"
          >
            <span
              class="makeStyles-iconStyle-11 MuiStepLabel-iconContainer css-vnkopk-MuiStepLabel-iconContainer"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiStepIcon-root css-1ktinih-MuiSvgIcon-root-MuiStepIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <circle
                  cx="12"
                  cy="12"
                  r="12"
                />
                <text
                  class="MuiStepIcon-text css-ighunu-MuiStepIcon-text"
                  dominant-baseline="central"
                  text-anchor="middle"
                  x="12"
                  y="12"
                >
                  2
                </text>
              </svg>
            </span>
            <span
              class="MuiStepLabel-labelContainer css-16ubnlw-MuiStepLabel-labelContainer"
            >
              <span
                class="makeStyles-labelStyle1-12 MuiStepLabel-label Mui-disabled css-gv1taf-MuiStepLabel-label"
              >
                Keywords
              </span>
            </span>
          </span>
        </div>
        <div
          class="MuiStepConnector-root MuiStepConnector-horizontal Mui-disabled css-j5w0w9-MuiStepConnector-root"
        >
          <span
            class="MuiStepConnector-line MuiStepConnector-lineHorizontal css-z7uhs0-MuiStepConnector-line"
          />
        </div>
        <div
          class="MuiStep-root MuiStep-horizontal makeStyles-step-19 css-1bw0nnu-MuiStep-root"
        >
          <span
            class="makeStyles-stepLabelRoot-10 MuiStepLabel-root MuiStepLabel-horizontal Mui-disabled css-ascpo7-MuiStepLabel-root"
          >
            <span
              class="makeStyles-iconStyle-11 MuiStepLabel-iconContainer css-vnkopk-MuiStepLabel-iconContainer"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiStepIcon-root css-1ktinih-MuiSvgIcon-root-MuiStepIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <circle
                  cx="12"
                  cy="12"
                  r="12"
                />
                <text
                  class="MuiStepIcon-text css-ighunu-MuiStepIcon-text"
                  dominant-baseline="central"
                  text-anchor="middle"
                  x="12"
                  y="12"
                >
                  3
                </text>
              </svg>
            </span>
            <span
              class="MuiStepLabel-labelContainer css-16ubnlw-MuiStepLabel-labelContainer"
            >
              <span
                class="makeStyles-labelStyle1-12 MuiStepLabel-label Mui-disabled css-gv1taf-MuiStepLabel-label"
              >
                Vehicles
              </span>
            </span>
          </span>
        </div>
        <div
          class="MuiStepConnector-root MuiStepConnector-horizontal Mui-disabled css-j5w0w9-MuiStepConnector-root"
        >
          <span
            class="MuiStepConnector-line MuiStepConnector-lineHorizontal css-z7uhs0-MuiStepConnector-line"
          />
        </div>
        <div
          class="MuiStep-root MuiStep-horizontal makeStyles-step-19 css-1bw0nnu-MuiStep-root"
        >
          <span
            class="makeStyles-stepLabelRoot-10 MuiStepLabel-root MuiStepLabel-horizontal Mui-disabled css-ascpo7-MuiStepLabel-root"
          >
            <span
              class="makeStyles-iconStyle-11 MuiStepLabel-iconContainer css-vnkopk-MuiStepLabel-iconContainer"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiStepIcon-root css-1ktinih-MuiSvgIcon-root-MuiStepIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <circle
                  cx="12"
                  cy="12"
                  r="12"
                />
                <text
                  class="MuiStepIcon-text css-ighunu-MuiStepIcon-text"
                  dominant-baseline="central"
                  text-anchor="middle"
                  x="12"
                  y="12"
                >
                  4
                </text>
              </svg>
            </span>
            <span
              class="MuiStepLabel-labelContainer css-16ubnlw-MuiStepLabel-labelContainer"
            >
              <span
                class="makeStyles-labelStyle1-12 MuiStepLabel-label Mui-disabled css-gv1taf-MuiStepLabel-label"
              >
                Sales Codes
              </span>
            </span>
          </span>
        </div>
        <div
          class="MuiStepConnector-root MuiStepConnector-horizontal Mui-disabled css-j5w0w9-MuiStepConnector-root"
        >
          <span
            class="MuiStepConnector-line MuiStepConnector-lineHorizontal css-z7uhs0-MuiStepConnector-line"
          />
        </div>
        <div
          class="MuiStep-root MuiStep-horizontal makeStyles-step-19 css-1bw0nnu-MuiStep-root"
        >
          <span
            class="makeStyles-stepLabelRoot-10 MuiStepLabel-root MuiStepLabel-horizontal Mui-disabled css-ascpo7-MuiStepLabel-root"
          >
            <span
              class="makeStyles-iconStyle-11 MuiStepLabel-iconContainer css-vnkopk-MuiStepLabel-iconContainer"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiStepIcon-root css-1ktinih-MuiSvgIcon-root-MuiStepIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <circle
                  cx="12"
                  cy="12"
                  r="12"
                />
                <text
                  class="MuiStepIcon-text css-ighunu-MuiStepIcon-text"
                  dominant-baseline="central"
                  text-anchor="middle"
                  x="12"
                  y="12"
                >
                  5
                </text>
              </svg>
            </span>
            <span
              class="MuiStepLabel-labelContainer css-16ubnlw-MuiStepLabel-labelContainer"
            >
              <span
                class="makeStyles-labelStyle1-12 MuiStepLabel-label Mui-disabled css-gv1taf-MuiStepLabel-label"
              >
                Dates
              </span>
            </span>
          </span>
        </div>
        <div
          class="MuiStepConnector-root MuiStepConnector-horizontal Mui-disabled css-j5w0w9-MuiStepConnector-root"
        >
          <span
            class="MuiStepConnector-line MuiStepConnector-lineHorizontal css-z7uhs0-MuiStepConnector-line"
          />
        </div>
        <div
          class="MuiStep-root MuiStep-horizontal makeStyles-step-19 css-1bw0nnu-MuiStep-root"
        >
          <span
            class="makeStyles-stepLabelRoot-10 MuiStepLabel-root MuiStepLabel-horizontal Mui-disabled css-ascpo7-MuiStepLabel-root"
          >
            <span
              class="makeStyles-iconStyle-11 MuiStepLabel-iconContainer css-vnkopk-MuiStepLabel-iconContainer"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiStepIcon-root css-1ktinih-MuiSvgIcon-root-MuiStepIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <circle
                  cx="12"
                  cy="12"
                  r="12"
                />
                <text
                  class="MuiStepIcon-text css-ighunu-MuiStepIcon-text"
                  dominant-baseline="central"
                  text-anchor="middle"
                  x="12"
                  y="12"
                >
                  6
                </text>
              </svg>
            </span>
            <span
              class="MuiStepLabel-labelContainer css-16ubnlw-MuiStepLabel-labelContainer"
            >
              <span
                class="makeStyles-labelStyle1-12 MuiStepLabel-label Mui-disabled css-gv1taf-MuiStepLabel-label"
              >
                LOPs Parts
              </span>
            </span>
          </span>
        </div>
        <div
          class="MuiStepConnector-root MuiStepConnector-horizontal Mui-disabled css-j5w0w9-MuiStepConnector-root"
        >
          <span
            class="MuiStepConnector-line MuiStepConnector-lineHorizontal css-z7uhs0-MuiStepConnector-line"
          />
        </div>
        <div
          class="MuiStep-root MuiStep-horizontal makeStyles-step-19 css-1bw0nnu-MuiStep-root"
        >
          <span
            class="makeStyles-stepLabelRoot-10 MuiStepLabel-root MuiStepLabel-horizontal Mui-disabled css-ascpo7-MuiStepLabel-root"
          >
            <span
              class="makeStyles-iconStyle-11 MuiStepLabel-iconContainer css-vnkopk-MuiStepLabel-iconContainer"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiStepIcon-root css-1ktinih-MuiSvgIcon-root-MuiStepIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <circle
                  cx="12"
                  cy="12"
                  r="12"
                />
                <text
                  class="MuiStepIcon-text css-ighunu-MuiStepIcon-text"
                  dominant-baseline="central"
                  text-anchor="middle"
                  x="12"
                  y="12"
                >
                  7
                </text>
              </svg>
            </span>
            <span
              class="MuiStepLabel-labelContainer css-16ubnlw-MuiStepLabel-labelContainer"
            >
              <span
                class="makeStyles-labelStyle1-12 MuiStepLabel-label Mui-disabled css-gv1taf-MuiStepLabel-label"
              >
                Summary
              </span>
            </span>
          </span>
        </div>
      </div>
      <div>
        <button
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium makeStyles-iconBtnNext-22 css-1vmeuix-MuiButtonBase-root-MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-17slstj-MuiSvgIcon-root"
            data-testid="NavigateNextRoundedIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M9.31 6.71c-.39.39-.39 1.02 0 1.41L13.19 12l-3.88 3.88c-.39.39-.39 1.02 0 1.41.39.39 1.02.39 1.41 0l4.59-4.59c.39-.39.39-1.02 0-1.41L10.72 6.7c-.38-.38-1.02-.38-1.41.01z"
            />
          </svg>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <p
          class="makeStyles-titleBtnNext-21"
          style="color: rgb(18, 41, 62);"
        >
          Next
        </p>
      </div>
    </div>
    <div
      class="makeStyles-contentContainer-4"
      id="123456"
    >
      <div
        class="makeStyles-content-6 makeStyles-contentWithCollapsedSidebar-7 makeStyles-contentWithStepper-8"
      >
        <div
          aria-hidden="true"
          class="MuiBackdrop-root css-1ji7omv-MuiBackdrop-root"
          role="loading"
          style="opacity: 0; visibility: hidden;"
        >
          <span
            class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorInherit css-62e83j-MuiCircularProgress-root"
            role="progressbar"
            style="width: 40px; height: 40px;"
          >
            <svg
              class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
              viewBox="22 22 44 44"
            >
              <circle
                class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
                cx="44"
                cy="44"
                fill="none"
                r="20.2"
                stroke-width="3.6"
              />
            </svg>
          </span>
        </div>
        <div
          style="margin-bottom: 25px;"
        >
          <div
            class="makeStyles-header1-31"
          >
            Request Title
          </div>
          <div
            class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary makeStyles-input-29 css-ffkbon-MuiInputBase-root-MuiInput-root"
          >
            <input
              class="MuiInput-input MuiInputBase-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
              inputmode="numeric"
              pattern="[0-9]*"
              placeholder="Request title"
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          class="makeStyles-container-40 MuiBox-root css-0"
        >
          <div
            class="MuiListItemButton-root MuiListItemButton-gutters MuiButtonBase-root css-1jni0wj-MuiButtonBase-root-MuiListItemButton-root"
            role="button"
            tabindex="0"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium makeStyles-customIcon-41 css-i4bv87-MuiSvgIcon-root"
              data-testid="ExpandLessRoundedIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M11.29 8.71 6.7 13.3c-.39.39-.39 1.02 0 1.41.39.39 1.02.39 1.41 0L12 10.83l3.88 3.88c.39.39 1.02.39 1.41 0 .39-.39.39-1.02 0-1.41L12.7 8.71c-.38-.39-1.02-.39-1.41 0z"
              />
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body1 css-8guo7j-MuiTypography-root"
            >
              Data sources
            </p>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </div>
          <div
            class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-entered css-pwcg7p-MuiCollapse-root"
            style="min-height: 0px;"
          >
            <div
              class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
            >
              <div
                class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
              >
                <div
                  class="MuiGrid-root MuiGrid-container css-13z16su-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-dz5dyu-MuiGrid-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body2 css-tfk0my-MuiTypography-root"
                    >
                      Required
                    </p>
                  </div>
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 css-1196aik-MuiGrid-root"
                  />
                </div>
                <div
                  class="MuiGrid-root MuiGrid-container css-13z16su-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-dz5dyu-MuiGrid-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body2 css-tfk0my-MuiTypography-root"
                    >
                      Optional
                    </p>
                  </div>
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 css-1196aik-MuiGrid-root"
                  />
                </div>
                <div
                  class="MuiGrid-root MuiGrid-container css-13z16su-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-dz5dyu-MuiGrid-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body2 css-tfk0my-MuiTypography-root"
                    >
                      Not Applicable
                    </p>
                  </div>
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 css-1196aik-MuiGrid-root"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="makeStyles-container-46 MuiBox-root css-0"
        >
          <div
            class="MuiBox-root css-0"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 makeStyles-stepTitle-47 css-16w2zvr-MuiTypography-root"
            >
              Step 
              4
               :
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 makeStyles-stepName-48 css-16w2zvr-MuiTypography-root"
            >
              Sales Codes
            </p>
          </div>
        </div>
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 rounded-lg overflow-hidden css-1ps6pg7-MuiPaper-root"
        >
          <div
            class="makeStyles-header-51"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 css-16w2zvr-MuiTypography-root"
            >
              Sales Code
            </p>
          </div>
          <div
            class="makeStyles-body-52"
          >
            <div
              class="MuiFormControl-root css-1nrlq1o-MuiFormControl-root"
            >
              <div
                class="MuiFormGroup-root MuiFormGroup-row px-5 css-qfz70r-MuiFormGroup-root"
                role="radiogroup"
              >
                <label
                  class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
                >
                  <span
                    class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root css-xbzm0a-MuiButtonBase-root-MuiRadio-root"
                  >
                    <input
                      class="PrivateSwitchBase-input css-1m9pwf3"
                      name="matchAllSalesCode.value"
                      type="radio"
                      value="MATCH_ALL_SALES_CODE"
                    />
                    <span
                      class="css-hyxlzm"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                        data-testid="RadioButtonUncheckedIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                        />
                      </svg>
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                        data-testid="RadioButtonCheckedIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                        />
                      </svg>
                    </span>
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </span>
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-16w2zvr-MuiTypography-root"
                  >
                    MATCH_ALL_SALES_CODE
                  </span>
                </label>
                <label
                  class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
                >
                  <span
                    class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root css-xbzm0a-MuiButtonBase-root-MuiRadio-root"
                  >
                    <input
                      class="PrivateSwitchBase-input css-1m9pwf3"
                      name="matchAllSalesCode.value"
                      type="radio"
                      value="MATCH_ANY_SALES_CODE"
                    />
                    <span
                      class="css-hyxlzm"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                        data-testid="RadioButtonUncheckedIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                        />
                      </svg>
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                        data-testid="RadioButtonCheckedIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                        />
                      </svg>
                    </span>
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </span>
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-16w2zvr-MuiTypography-root"
                  >
                    MATCH_ANY_SALES_CODE
                  </span>
                </label>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 pl-4 mt-4 makeStyles-contentBody-54 css-16ysvd0-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 my-auto pl-5 css-wpkret-MuiGrid-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-16w2zvr-MuiTypography-root"
                >
                  1
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 pr-4 makeStyles-salesCodeLookup-56 css-fyezmf-MuiGrid-root"
              >
                <div
                  class="MuiAutocomplete-root css-16awh2u-MuiAutocomplete-root"
                >
                  <div
                    class="MuiFormControl-root MuiFormControl-fullWidth makeStyles-root-58 MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
                  >
                    <div
                      class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-sizeSmall MuiInputBase-adornedEnd MuiAutocomplete-inputRoot css-1mxjs2u-MuiInputBase-root-MuiOutlinedInput-root"
                    >
                      <input
                        aria-autocomplete="list"
                        aria-expanded="false"
                        aria-invalid="false"
                        autocapitalize="none"
                        autocomplete="off"
                        class="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputSizeSmall MuiInputBase-inputAdornedEnd MuiAutocomplete-input MuiAutocomplete-inputFocused css-19qh8xo-MuiInputBase-input-MuiOutlinedInput-input"
                        id="free-solo-demo"
                        role="combobox"
                        spellcheck="false"
                        type="text"
                        value=""
                      />
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                        data-testid="SearchIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                        />
                      </svg>
                      <fieldset
                        aria-hidden="true"
                        class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                      >
                        <legend
                          class="css-hdw1oc"
                        >
                          <span
                            class="notranslate"
                          >
                            ​
                          </span>
                        </legend>
                      </fieldset>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 relative css-wpkret-MuiGrid-root"
              >
                <button
                  class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall MuiButton-containedSizeSmall MuiButtonBase-root rounded-full bg-[#F96A6A] w-5 h-5 min-w-0 absolute top-1/2 left-1/2 makeStyles-deleteButton-55 css-t8red9-MuiButtonBase-root-MuiButton-root"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-startIcon MuiButton-iconSizeSmall css-y6rp3m-MuiButton-startIcon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                      data-testid="ClearIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
            <button
              class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeLarge MuiButton-textSizeLarge MuiButtonBase-root ml-4 pl-0 hover:bg-transparent capitalize font-[700] mt-10 css-1lalbl4-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-startIcon MuiButton-iconSizeLarge css-htszrh-MuiButton-startIcon"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-ohmddv-MuiSvgIcon-root"
                  data-testid="AddCircleIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm5 11h-4v4h-2v-4H7v-2h4V7h2v4h4v2z"
                  />
                </svg>
              </span>
              Add New Row
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="makeStyles-btnContainer-1 MuiBox-root css-0"
        >
          <button
            class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root makeStyles-btnPrevious-2 css-1i3jxbp-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-startIcon MuiButton-iconSizeMedium css-1d6wzja-MuiButton-startIcon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="SkipPreviousRoundedIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 6c.55 0 1 .45 1 1v10c0 .55-.45 1-1 1s-1-.45-1-1V7c0-.55.45-1 1-1zm3.66 6.82 5.77 4.07c.66.47 1.58-.01 1.58-.82V7.93c0-.81-.91-1.28-1.58-.82l-5.77 4.07c-.57.4-.57 1.24 0 1.64z"
                />
              </svg>
            </span>
            Previous
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
          <button
            class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root makeStyles-btnNext-3 css-13itokw-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            Skip
            <span
              class="MuiButton-endIcon MuiButton-iconSizeMedium css-9tj150-MuiButton-endIcon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="SkipNextRoundedIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="m7.58 16.89 5.77-4.07c.56-.4.56-1.24 0-1.63L7.58 7.11C6.91 6.65 6 7.12 6 7.93v8.14c0 .81.91 1.28 1.58.82zM16 7v10c0 .55.45 1 1 1s1-.45 1-1V7c0-.55-.45-1-1-1s-1 .45-1 1z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
  </main>
</div>
`;
